www.chuyu.me/es.xml

```diff
@@ -27,7 +27,7 @@
   <String Src="%GreenDes%" Dst="Portable, Libre, seguro, y sin anuncios."/>
   <String Src="%AboutChuyu%" Dst="Acerca del equipo Chuyu"/>
   <String Src="%AboutChuyuDes%" Dst="Somos perfeccionistas y bastante obsesivos para crear una herramienta que sea ligera, fácil de usar, directa, y sin molestias. Todovia no lo hemos logrado, pero no nos dimos por vencidos mientras realizamos un sueño con nuestras propias manos haciendo Dism++: una pequeña herramienta sin fines de lucro y construida con pura persistencia y entusiasmo."/>
-  <String Src="%QQ%" Dst="Grupo oficial"/>
+  <String Src="%QQ%" Dst="Grupos oficiales"/>
```

Comment:
Ves otra vez estamos con lo mismo, no decías que no ibas a cambiar nada más sin comentarlo antes.
Not is "Grupos oficiales", is  "Grupo oficial" o "Grupo QQ". The original is "QQ Group", not "QQ Groups".

Tú palabra se la lleva el viento, veo que no tienes palabra, no cambies nada más, no puedo estar detrás de todas las malas traducciones que haces. Intento ser lo más correcto que puedo, pero